mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -28,7 +29,8 @@
     hint = "smart-testing")
 class SmartTestingMavenConfigurer extends AbstractMavenLifecycleParticipant {
 
-    private static final Logger logger = Logger.getLogger();
+    @Requirement
+    public static org.codehaus.plexus.logging.Logger LOGGER;
```

Comment:
@MatousJobanek @bartoszmajsak, the injected Logger always returns a null value. I am unable to detect the issue. Is there something basic that I am missing here for the DI to work ?

Reply:
It cannot be `static` in this case. Injection works with instance fields.